spec.bs

```diff
@@ -496,6 +522,20 @@ scope=] |debugScope| and an optional [=debug details=] or null
         set |entry|'s [=contribution cache entry/debug details=] to
         |debugDetails|.
 
+To <dfn>determine if a null report should be sent</dfn> given a [=pre-specified
```

Comment:
Shall we rename the algorithm to something like `determine if a report should be sent deterministically` as it' not necessarily a null report?

Reply:
Makes sense! Done